HHS/Head-Start-TTADP
# [TTAHUB-1826] Added permissions for edit and create on training report landing

## Description of change

Added permissions for edit and create on training report landing.

## How to test

- Create a user with only read only permissions for training reports. You should not have the ability to create or edit a event or session.
- Create a user with read write permission for training reports. You should have the ability to create and edit event and session reports.
- The view option should be available for both paths.


## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1826


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
